content/docs/apps/managing-teammates.md

```diff
@@ -62,3 +62,95 @@ Decide which roles to give them. For a complete list of roles, see [the Cloud Fo
     cf set-org-role <email> <org> OrgAuditor
     ```
 {{% /commandline %}}
+
+## Removing a teammate
+
+If you are an Org Manager or Space Manager, you can remove teammates from orgs and spaces respectively.
+
+{{% dashboard %}}
```

Comment:
@fureigh Org Managers are currently the only people who can invite users. The reference on line 22 is still correct. All users have access to the dashboard. There is a ticket that Adam is working on that would require a change to the documentation. In regards to my PR that adds the remove user functionality/notice, this is correct.